src/main/java/com/online/cinema/Main.java

```diff
@@ -70,39 +72,39 @@ public static void main(String[] args) {
         authenticationService.register("user1@gmail.com", "pass");
         authenticationService.register("user2@gmail.com", "pass");
         User user = userService.findByEmail("user1@gmail.com").get();
-        System.out.println(user.toString());
+        logger.info(user.toString());
         try {
-            System.out.println(authenticationService.login("user2@gmail.com", "pass").toString());
+            logger.info(authenticationService.login("user2@gmail.com", "pass").toString());
         } catch (AuthenticationException e) {
-            e.printStackTrace();
+            logger.error(e.getMessage(), e);
```

Comment:
```suggestion
            logger.warn("Login failed: " + e.getMessage(), e);
```
nothing really bad happened here, `warn` level is enough